bors.toml

```diff
@@ -1,6 +1,7 @@
 status = [
-  "buildkite/cardano-wallet",
-  "ci/hydra-build:required",
+  "buildkite/cardano-wallet"
+  , "buildkite/cardano-wallet-checks-compiled-commit"
+  , "buildkite/cardano-wallet-checks-compiled-merge"
```

Comment:
👍 Was about to comment we should also have these.

The "only run integration tests when pressing the button" might be inconvenient though 🤔 Now that we have bors, I suspect we should configure them to run automatically on the `bors/staging` or `bors/trying` branches

Reply:
Yes fixing that is part of this PR, the review is only functional to the work I will reiterate the request before merging, thanks